tcms/rpc/api/auth.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=import-outside-toplevel
```

Comment:
this also doesn't look right, you should moving the offending import at toplevel

Reply:
needs to be removed. this file needs more attention